packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -75,18 +75,20 @@ export class QueryRunnerArgsFactory {
       case ResolverArgsType.FindDuplicates:
         return {
           ...args,
-          id: await this.overrideValueByFieldMetadata(
-            'id',
-            (args as FindDuplicatesResolverArgs).id,
-            fieldMetadataMap,
-          ),
-          data: await this.overrideDataByFieldMetadata(
-            (args as FindDuplicatesResolverArgs).data,
-            options,
-            fieldMetadataMap,
-            { shouldBackfillPosition: false },
+          ids: (await Promise.all(
+            (args as FindDuplicatesResolverArgs).ids?.map((id) =>
+              this.overrideValueByFieldMetadata('id', id, fieldMetadataMap),
```

Comment:
Just realised that overrideValueByFieldMetadata is async and I'm not sure why 🤔

Reply:
I have no idea why either 🤔